### PR TITLE
Image Attribute support added for ['style','alt','width','height']

### LINKED ISSUE #6: Image attributes stripped in editor
If an image has any attributes other than "src", "class" and "style" this is removed when rendered in the editor. After investigating the code, this appears to be caused by the following lines of code in "ngx-quill-upload\__ivy_ngcc__\fesm2015\ngx-quill-upload.js"

```
const blotPath = 'blots/block/embed';
const BlockEmbed = Quill.import(blotPath);
class ImageBlot extends BlockEmbed {
```
Changing the blotPath variable to "formats/image" resolves this issue.

### PR DESCRIPTION
**Pull request by ayush013**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,9 +37,9 @@
     "@angular/compiler": "^10.0.11",
     "@angular/compiler-cli": "^10.0.11",
     "@angular/core": "^10.0.11",
-    "ng-packagr": "^10.1.0"
+    "ng-packagr": "^10.1.0",
```
Comment (shubh9457) [inline package.json]: @ayush013 Can you remove all the carat signs from all the packages?

File: src/blots/ImageBlot/index.ts [hunk 2]
```diff
@@ -1,8 +1,22 @@
...
 import { Constants } from '../../utils';
 
-const blotPath = 'blots/block/embed';
-const BlockEmbed = Quill.import(blotPath);
+const blotPath = 'formats/image';
+const BlockEmbed: EmbedBlot = Quill.import(blotPath);
+
+interface EmbedBlot {
+  new(...args: any[]): EmbedBlot;
+  domNode: any;
```
Comment (shubh9457) [inline src/blots/ImageBlot/index.ts]: @ayush013 Can we have more strict type for domNode?

**Review by shubh9457 — CHANGES_REQUESTED** [review]

File: src/blots/ImageBlot/index.ts (hunk 2 above)
Comment (ayush013) [reply src/blots/ImageBlot/index.ts]: Sure, `HTMLImageElement` will do

File: package.json (hunk 1 above)
Comment (ayush013) [reply package.json]: Sure

**Review by shubh9457 — APPROVED** [review]

**Pull request merged**